tasks/ceph_manager.py

```diff
@@ -793,7 +793,11 @@ def tmp(x):
         pools = self.list_pools()
         self.pools = {}
         for pool in pools:
-            self.pools[pool] = self.get_pool_property(pool, 'pg_num')
+            # we may race with a pool deletion; ignore failures here
+            try:
+                self.pools[pool] = self.get_pool_property(pool, 'pg_num')
+            except:
+                pass
```

Comment:
I'd recommend at least logging something here, or else you'll never know when this is happening. Unless you really don't want to know :)